proto/sentry_protos/snuba/v1/examples.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime
+from sentry_protos.snuba.v1.endpoint_time_series_pb2 import (
+    TimeSeriesRequest,
+    AttributeAggregation,
+    TimeSeries,
+    TimeSeriesResponse,
+    Function,
+)
+from sentry_protos.snuba.v1.trace_item_attribute_pb2 import AttributeKey, AttributeValue
+from sentry_protos.snuba.v1.endpoint_trace_item_attributes_pb2 import (
+    TraceItemAttributeNamesRequest,
+    TraceItemAttributeNamesResponse,
+    TraceItemAttributeValuesRequest,
+    TraceItemAttributeValuesResponse,
+)
+from sentry_protos.snuba.v1.request_common_pb2 import RequestMeta, TraceItemName
+from google.protobuf.timestamp_pb2 import Timestamp
+
+
+COMMON_META = RequestMeta(
+    project_ids=[1, 2, 3],
+    organization_id=1,
+    cogs_category="something",
+    referrer="something",
+    start_timestamp=Timestamp(seconds=int(datetime(2024, 4, 20, 16, 20).timestamp())),
+    end_timestamp=Timestamp(seconds=int(datetime(2024, 4, 20, 17, 20).timestamp())),
+    trace_item_name=TraceItemName.TRACE_ITEM_NAME_EAP_SPANS,
+)
+
+
+def test_example_time_series():
```

Comment:
@wmak thank you for the prompt to add examples. I will start adding them here as I do them. Also invite you to try and do your own as well and see if the mental model breaks anywhere